etl/steps/data/garden/artificial_intelligence/2023-07-25/cset.meta.yml

```diff
@@ -0,0 +1,161 @@
+    url: https://eto.tech/tou/
+  sources:
+  - name: Center for Security and Emerging Technology (2023)
+    url: https://cat.eto.tech/
+    date_accessed: '2023-07-25'
+    publication_year: 2023
+    published_by: Emerging Technology Observatory Country Activity Tracker, Artificial Intelligence (Center for Security and
+      Emerging Technology, 2023)
+tables:
+  cset:
+    variables:
+      disclosed_investment:
+        title: Disclosed investment
+        description:  Investments in AI-related companies across various industries where the investment value has been publicly disclosed.
+        unit: US dollars
+        short_unit: $
+        display:
+          numDecimalPlaces: 0
+          name: Disclosed
+
+      investment_estimated:
+        title: Estimated investment
+        description:  Estimated value of investments in AI-related companies across various industries when exact investment amounts aren't publicly disclosed. The estimate is based on the median value of similar investments from Crunchbase.
+        unit: US dollars
+        short_unit: $
+        display:
+          numDecimalPlaces: 0
+          name: Estimated
+
+      disclosed_investment_summary:
+        title: Total disclosed investment
+        description: Investments in AI-related companies across various industries where the investment value has been publicly disclosed.
+        unit: US dollars
+        short_unit: $
+        display:
+          numDecimalPlaces: 0
+          name: Disclosed
+
+      estimated_investment_summary:
+        title: Total estimated investment
+        description: >
+          Estimated value of investments in AI-related companies across various industries when exact investment amounts aren't publicly disclosed. The estimate is based on the median value of similar investments from Crunchbase.
+        unit: US dollars
+        short_unit: $
+        display:
+          numDecimalPlaces: 0
+          name: Estimated
+
+      num_patent_applications:
+        title: Patent applications
+        description: Yearly total number of patent applications.
+        unit: applications
+        display:
+          numDecimalPlaces: 0
+          name: Applications
+
+      num_patent_granted:
+        title: Patent applications granted
+        description: Yearly total number of patents granted.
+        unit: granted applications
+        display:
+          numDecimalPlaces: 0
+          name: Granted
+
+      num_patent_applications_summary:
+        title: Total patent applications
+        description: Total number of patent applications between 2010-2020.
+        unit: applications
+        display:
+          numDecimalPlaces: 0
+          name: Applications
+
+      num_patent_granted_summary:
+        title: Total patent applications granted
+        description: Total number of patents granted between 2010-2020.
+        unit: granted applications
+        display:
+          numDecimalPlaces: 0
+          name: Granted
```

Comment:
Perhaps making the display name slightly more specific would be useful, so here 'Patents granted'.

Reply:
actually I need to remove this altogether because I'd be expanding everything in grapher. Do you by any chance know if the only way to assign custom names for the variables that are created based on the index is manually in grapher?

Reply:
I have started doing it in the garden step like with the add_metadata() and create_variable_metadata() functions [here](https://github.com/owid/etl/blob/fabe2f1ea52276824755976eaf3fc7debda8fbca/etl/steps/data/garden/who/2023-08-01/mortality_database.py). I think you could also do something similar in the grapher step?

Reply:
hey this is pretty cool! I've tried to do something similar for my education datasets but I don't think it will work in the grapher step without modifying some hardcoded functions :( I think Mojmir will take a look at whether it's possible though and I'll let you know if we find a way in case this will also be useful for you :)

Reply:
That would be great - thanks!